### PR TITLE
update file

### PR DESCRIPTION
**Pull request by ngtrongnguyen**

### CODE REVIEW

**Comment by Lanng21** [comment]
pls split file changes, 1 PR no more than 10 change files :(( you can split the image files into another PR
and attached screenshot in each PR

File: src/components/nav.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<div class="header-navbar">
+    <div class="nav-left">
+        <div class="nav-logo">
+            <div class="nav-login-inside"></div>
+        </div>
+        <h1 class="nav-h1text" >PRC</h1>
+    </div>
+    <div class="nav-middle">
+        <ul class="nav-item">
+            <li class="nav-items">Home</li>
```
Comment (Lanng21) [inline src/components/nav.html:11]: U can using the tag for loop item

File: src/layouts/header.html [hunk 2]
```diff
@@ -1 +1,27 @@
-<h2>hi</h2>
+<header class="header">
+    <include src="./components/nav.html"></include>
+   
+    <div class="main-header">
+        <article class="main-text">
+            <h1 class="main-texth1">Rule the world as a Professional Real Estate Consultant</h1>
+           <div class="main-button">
+                <button class="main-btn">Get started</button>
```
Comment (Lanng21) [inline src/layouts/header.html:8]: i think u can move button to component for easy reused

File: src/layouts/header.html [hunk 3]
```diff
@@ -1 +1,27 @@
...
+                <button class="main-btn">Learn more</button>
+           </div>
+           <div class="main-input">
+                 <input type="radio" name="gender" value="female">
```
Comment (Lanng21) [inline src/layouts/header.html:12]: same

File: src/components/feature.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            </figure>
+        </div>
+        <div class="feature-imggroup">
+            <figure>
+                <img src="../assets/img/Rectangle 7.png" alt="" class="feature-imgitem image2">
+            </figure>
+        </div>
+        <div class="feature-imggroup">
+            <figure>
+
```
Comment (Lanng21) [inline src/components/feature.html:19]: break line

File: src/components/feature.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        <div class="feature-imggroup">
+            <figure>
+                
+                <img src="../assets/img/Rectangle 8.png" alt="" class="feature-imgitem" >
+                <img src="../assets/img/Rectangle 10.png" alt="" class="feature-imgitem">
+            </figure>
+        </div>
+    </div>
+
+</div>
```
Comment (Lanng21) [inline src/components/feature.html:26]: end of line

File: src/stylesheets/component/_mainheader.scss [hunk 6]
```diff
@@ -0,0 +1,85 @@
+
+.main-header{
+    width: 1043px;
```
Comment (Lanng21) [inline src/stylesheets/component/_mainheader.scss:3]: Why do you have to set the width height here?
Don't fix width and height in pixels that will make it difficult for you when you're responsive

File: src/stylesheets/component/_mainheader.scss [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    border-bottom-right-radius: 60%;
+/* Brand color/Secondary */
+
+    background: #1E3446;
+}
+.main-texth1{
+    position: relative;
+    top: 286px;
+    left: 104px;
+    width: 684px;
```
Comment (Lanng21) [inline src/stylesheets/component/_mainheader.scss:16]: same

File: src/stylesheets/component/_mainheader.scss [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+    font-family: 'Quicksand';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 56px;
+    line-height: 72px;
+/* or 129% */
+
+/* Pure White */
+
+    color: #FFFFFF;
```
Comment (Lanng21) [inline src/stylesheets/component/_mainheader.scss:30]: can u defind it in `_variable.scss`?I see it being reused everywhere

File: src/stylesheets/component/_mainheader.scss [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+    display: flex;
+    align-items: center;
+    text-align: center;
+
+   
+    bottom: 0;
+   
+    text-decoration: none;
+    color: #F58634 ;
```
Comment (Lanng21) [inline src/stylesheets/component/_mainheader.scss:60]: same

File: src/stylesheets/component/_nav.scss [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+   
+}
+.nav-logo{
+    position: relative;
+    width: 32px;
+    height: 32px;
+    left: 0px;
+    top: 57px;
+    border: 9px solid;
+
```
Comment (Lanng21) [inline src/stylesheets/component/_nav.scss:47]: break line

File: src/stylesheets/layout/_header.scss [hunk 11]
```diff
@@ -0,0 +1,14 @@
+@forward '../component/index';
+
+#root{
```
Comment (Lanng21) [inline src/stylesheets/layout/_header.scss:3]: u should move it to base folder

File: src/stylesheets/main.scss [hunk 12]
```diff
@@ -1,3 +1,7 @@
+@use "layout";
+@use "base";
+@use "pages";
 h1 {
```
Comment (Lanng21) [inline src/stylesheets/main.scss:4]: why u defind here??

File: src/stylesheets/component/_nav.scss [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+}
+.nav-h1text{
+    position: relative;
+    left: 70px;
+    color: #FFFFFF;
+}
+
+
+.nav-items--regis{
+    color: #F58634;
```
Comment (Lanng21) [inline src/stylesheets/component/_nav.scss:78]: color here

**Comment by Lanng21** [comment]
Don't fix width and height in pixels that will make it difficult for you when you're responsive

**Comment by Lanng21** [comment]
and u should defind variable for color

File: src/stylesheets/component/_mainheader.scss [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+.main-texth1{
+    position: relative;
+    top: 286px;
+    left: 104px;
+    width: 684px;
+    height: 216px;
+
+/* Heading 1 */
+
+    font-family: 'Quicksand';
```
Comment (haunguyen15-tt) [inline src/stylesheets/component/_mainheader.scss:21]: Can u move this to another place to reuse it? Something like `typography.scss` and put it into `base` folder

File: src/components/course.html [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+            <i class="fa-sharp fa-solid fa-star"></i>
+            <i class="fa-sharp fa-solid fa-star"></i>
+            <span>(5,908)</span>
+        </div>
+        <div class="course-btn">
+            <button class="btn-courseitem">View course</button>
+        </div>
+    </div>
+
+</div>
```
Comment (Lanng21) [inline src/components/course.html:65]: end of line

File: src/components/feature.html [hunk 16]
```diff
@@ -0,0 +1,26 @@
+<div class="main-feature">
+    <span>Features</span>
+    <h3>We Provide the Best Property for you</h3>
+    <div class="feature-img">
+        <div class="feature-imggroup">
```
Comment (Lanng21) [inline src/components/feature.html:5]: same. u should move it to component to reuse it. I see it being reused everywhere

**Pull request closed without merging**